.circleci/config.yml

```diff
@@ -287,33 +268,35 @@ workflows:
                 - dev
                 - stage
                 - test
-      - deploy-test:
+      - deploy:
           filters:
             branches:
               only:
                 - dev
                 - stage
-                - test
+                - prod
           requires:
             - build-theme
-      - cypress:
+      - build-static:
           filters:
             branches:
               only:
                 - dev
                 - stage
                 - test
+                - prod
           requires:
-            - deploy-test
-      - deploy:
+            - build-theme
+      - cypress:
           filters:
             branches:
               only:
+                - test
                 - dev
                 - stage
                 - prod
```

Comment:
We do not want tests to be run on `prod` We should have all tests done before deploy to PROD.